scss/catppuccin/mocha.scss

```
@@ -11,6 +11,10 @@ $hover: (
   hover-fg: #89b4fa,
 );
 
+$mark: (
+  search-mark-bg: #fab387,
+);
```

Stylelint (17.3.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)